modules/android/examples/codesnippet_collection.java

```diff
@@ -2746,18 +2760,25 @@ public void activeReplicatorExample(Set<Collection> collections)
         thisReplicator = repl;
         thisToken = token;
 
-        // end::p2p-act-rep-start-full[]
         // end::p2p-act-rep-func[]
     }
 
+    public void replicatorCopyConfig(ReplicatorConfiguration replicatorConfiguration, Replicator repl) {
+        // tag::p2p-act-rep-start-full[]
+        ReplicatorConfiguration replConfig = new ReplicatorConfiguration(replicatorConfiguration); // <.>
+        repl.start(); // <.>
+        // end::p2p-act-rep-start-full[]
+    }
+
     public void replicatorSimpleExample(Set<Collection> collections) throws URISyntaxException {
         // tag::replicator-simple[]
         Endpoint theListenerEndpoint
             = new URLEndpoint(new URI("wss://10.0.2.2:4984/db")); // <.>
 
+        Set<CollectionConfiguration> collConfig = CollectionConfiguration.fromCollections(collections);
```

Comment:
Prefer the plural name such as collConfigs as it's a Set. Please check the other code snippets as well.